aldryn_jobs/views.py

```diff
@@ -6,12 +6,13 @@
 from django.http import Http404
 from django.shortcuts import redirect
 from django.utils.translation import (
-    ugettext_lazy as _, get_language_from_request
+    ugettext as _, get_language_from_request
```

Comment:
One trick I've seen folks use, which i Iike, is use a double underscore (`__`) for the throwaway var. Not sure how linting tools handle it - most will recognize that the single underscore is throwaway and will not warn about variable assigned and unused.